addons/web/static/tests/views/fields/statusbar_time_field_tests.js

```diff
@@ -0,0 +1,112 @@
+/** @odoo-module **/
+
+import {click, getFixture} from "@web/../tests/helpers/utils";
```

Comment:
don't forget to run the prettier on the js diff. We have tooling that you can easily enable: ./addons/web/tooling/enable.sh
It automatically lints/prettifies javascript code in white listed addons (including web) when you generate a commit.

Also, a lot of commits can be squashed imho. Talking about the StatusBarTimeField (which is my major concern here), a single commit including it's last version (js, css, xml + tests) would be fine.